cpu/sam0_common/Makefile.include

```diff
@@ -30,6 +30,7 @@ CFLAGS += -DDONT_USE_PREDEFINED_PERIPHERALS_HANDLERS
 LINKER_SCRIPT ?= cortexm.ld
 
 INCLUDES += -I$(RIOTCPU)/sam0_common/include
+INCLUDES += $(if $(filter sam0_eth,$(USEMODULE)),-I$(RIOTCPU)/sam0_common/sam0-eth/)
```

Comment:
Why not `ifneq ... endif`? It is IMO easier to read and easier to extend that way.

Reply:
Fixed.